lib/default-file-icons.coffee

```diff
@@ -0,0 +1,23 @@
+fs = require 'fs-plus'
+path = require 'path'
+
+class DefaultFileIcons
+  getIconForPath: (filePath) ->
```

Comment:
Yeah, I like that idea. Maybe `iconClassForPath`? It seems like we usually don't put `get` in the method name when it has the pattern `{thing}For{OtherThing}`.